### PR TITLE
Adds `run-redmine.sh` script as the container command

### PR DESCRIPTION
**Pull request by sameersbn**
**Description of the change**

- Adds `run-redmine.sh` script to replace `nami start redmine` as the container command

**Benefits**

 - Speeds up container launch
 - Lower RAM utilization

**Possible drawbacks**

**Applicable issues**

**Additional information**

 - Since `nami` is still used for the initialization tasks, the RAM utilization during the initialization phase is still higher then required.

### CODE REVIEW

File: 3/Dockerfile [hunk 1]
```diff
@@ -17,6 +20,16 @@ RUN bitnami-pkg install git-2.10.1-1 --checksum 454e9eb6fb781c8d492f9937439dcdfc
 # Install redmine
 RUN bitnami-pkg unpack redmine-3.3.2-0 --checksum 7488ba7ec4b015add6ac0293de8267ff02fc1a4070c6774a934800794ff8ee16
 
+# Install gosu
```
Comment (jotadrilo) [inline 3/Dockerfile]: I added some concerns in https://github.com/bitnami/bitnami-docker-ghost/pull/49 that apply here too

File: 3/rootfs/run-redmine.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+. /opt/bitnami/base/helpers
+
+USER=redmine
+RAILS_ENV=production
+PID_FILE=/opt/bitnami/redmine/tmp/pids/redmine.pid
+LOG_FILE=/opt/bitnami/redmine/logs/production.log
+
+mkdir -p /opt/bitnami/redmine/tmp/pids
+chown ${USER}: -R /opt/bitnami/redmine/tmp
+chmod -R 1777 /opt/bitnami/redmine/tmp
```
Comment (jotadrilo) [inline 3/rootfs/run-redmine.sh]: Why do you need the usage of the sticky bit here?

File: 3/rootfs/run-redmine.sh [hunk 3]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -e
+. /opt/bitnami/base/functions
+. /opt/bitnami/base/helpers
+
+USER=redmine
+RAILS_ENV=production
+PID_FILE=/opt/bitnami/redmine/tmp/pids/redmine.pid
+LOG_FILE=/opt/bitnami/redmine/logs/production.log
+
+mkdir -p /opt/bitnami/redmine/tmp/pids
```
Comment (jotadrilo) [inline 3/rootfs/run-redmine.sh]: We may want to move the permissions logic to the Redmine module if needed. However, I would like to understand why these changes are required here.

Comment (sameersbn) [inline 3/rootfs/run-redmine.sh]: yes, this should be moved to the `redmine` module.
This is required so that passenger (start as `redmine` user) can create the pid file.

File: 3/rootfs/run-redmine.sh (hunk 2 above)
Comment (sameersbn) [inline 3/rootfs/run-redmine.sh]: Without the sticky bit set, any user with write and execute permissions for the directory can rename or delete contained files, regardless of the file's owner. This is normally set on `/tmp` directory and I would treat the applications `tmp` directory in the same way.
